Protos/SwiftProtobufTests/test_messages_proto3.proto

```diff
@@ -60,44 +60,65 @@ message TestAllTypesProto3 {
   }
 
   // Singular
-  int32 optional_int32 = 1;
```

Comment:
This message and hence these fields are used for a bunch of tests, so while things pass, it likely means exact what was getting testing changes for a bunch of other things.  Probably makes sense to instead use a dedicated message (or file with message) for these specific tests so there's less chance of changing something else.